lib/movie.dart

```diff
@@ -0,0 +1,44 @@
+import 'package:flutter/material.dart';
+
+class Movie {
+  final Image backdrop;
+  final List<String> genres;
+  final double voteAverage;
+  final Image poster;
+  final String overview;
+  final String originalTitle;
+  final DateTime releaseDate;
+  final String title;
+  int likes = 0;
+
+  Movie(
+    this.title,
+    this.originalTitle,
+    this.genres,
+    this.poster,
+    this.backdrop,
+    this.releaseDate,
+    this.voteAverage,
+    this.overview,
+  );
+
+  String get formattedReleaseDate =>
+      '${releaseDate.day}/${releaseDate.month}/${releaseDate.year}';
+
+  void addLike() async => likes++;
+
+  factory Movie.defaultMovie() => Movie(
+        'The Super Mario Bros. Movie',
+        'The Super Mario Bros. Movie',
+        ['Animation', 'Family', 'Adventure', 'Fantasy', 'Comedy'],
+        Image.network(
+            'https://image.tmdb.org/t/p/w500/qNBAXBIQlnOThrVvA6mA2B5ggV6.jpg'),
+        Image.network(
+            "https://image.tmdb.org/t/p/w500/9n2tJBplPbgR2ca05hS5CKXwP2c.jpg"),
```

Comment:
I'd leave `Movie` as a data class only, not having any reference to widgets at all. You should leave both `poster` and `backdrop` fields as string fields. Then it's up to the presentation layer to decide how to present that information

Reply:
Done